glowing_waffles/visualization/aij_plots.py

```diff
@@ -0,0 +1,49 @@
+import matplotlib.pyplot as plt
+
+__all__ = ['seeing_plot']
+
+def seeing_plot(raw_radius, raw_counts, binned_radius, binned_counts, HWHM, plot_title=''):
+    """
+    Show a seeing plot for data from an image w/ radius on the x axis and counts (ADU) on the y axis
```

Comment:
Some really minor suggestions: write out "with" and a period on the end. I would take the "(ADU)" out since it may or may not be in ADU, depending on what the user has passed in.